src/domain/hanke/edit/reducer.ts

```diff
@@ -39,10 +39,12 @@ const formSlice = createSlice({
       if (payload) {
         state.status = 'ok';
         state.hankeDataDraft = payload;
+        toast.success('Tallennettii onnistuneesti');
```

Comment:
Side-efektit pitää laittaa thunkin sisään: https://stackoverflow.com/questions/36016336/why-does-a-redux-reducer-have-to-be-side-effect-free